File: front-end/excelsports/src/model/Exercise.ts
export class Exercise {
    id:number;
    trainingSessionsId: number;
    category: string;
    exerciseName: string;
    currentValue: number;
    goalId?:number;

  constructor(id:number,sessionId: number, category: string, exerciseName: string, value: number) {
      this.id=id;
    this.trainingSessionsId = sessionId;
    this.category = category;
    this.exerciseName = exerciseName;
    this.currentValue = value;
  }

  }
  

File: backend/exercise-service/src/main/java/com/ust/excelsports/model/Exercise.java

package com.ust.excelsports.model;

import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Entity
@AllArgsConstructor
@NoArgsConstructor
public class Exercise {

    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private int id;

    private String exerciseName;

    private int currentValue;

    private String category;

    private int trainingSessionsId;

    private int goalId;


}
